docs/contributing/overview.md

```diff
@@ -1,53 +1,66 @@
 
-### Discussions:
+## Discussions
 
-- We have the following discussion topics: 
+- We have the following discussion topics:
     1. [Announcements](https://github.com/aquasecurity/tracee/discussions/categories/announcements): One way communication from the team to the community. Consider this like our mini blog
     1. [Questions and Help](https://github.com/aquasecurity/tracee/discussions/categories/questions-and-help): For help and support. Consider this similar to StackOverflow.
     1. [Development](https://github.com/aquasecurity/tracee/discussions/categories/development): For discussing potential features, and collaborating on their design.
 
-### Issues:
-
-1. Every issue needs to be actionable and assignable. Consider the scope of the issue if assigned to one person, and break down if necessary.
-1. Be clear and definitive when composing issues. For bug reports, include detailed error messages and environment description. For features, include a clear scope and acceptance criteria.
-1. Since we have different projects under the same monorepo, use labels to denote areas that the issue relates to:
-    1. `tracee`
-    1. `tracee-ebpf`
-    1. `tracee-rules`
-    1. `signatures`
-    1.  If non of the labels is relevant don't add any (usually for top-level issues)
-1. We use the following labels to describe the type of issue:
-    1. `bug`
-    1. `good-first-issue`
-1. Self-assign or request assignment for issues you intend to work on. Don't work on an issue assigned to someone else without checking with them first and reassigning.
+## Issues
+
+1. **Actionable and Assignable:** Every issue must be clear, actionable, and assignable to a specific person.  Break down large issues into smaller, more manageable tasks.
+2. **Clear and Definitive Descriptions:** Be precise in your issue descriptions:
+    - **Bug Reports:** Include the following:
+        - Detailed steps to reproduce the bug.
+        - The complete error message.
+        - Your operating system, Tracee version, and any other relevant environment details.
+    - **Feature Requests:** Define:
+        - A clear scope for the feature.
+        - Specific acceptance criteria that will be used to determine if the feature is complete.
+3. **Issue Labels:** We use these labels to categorize and track issues:
+    - `area/...` (e.g., `ebpf`, `events`): Specifies the area of Tracee affected by the issue.
+    - `kind/...` (e.g., `bug`, `chore`, `enhancement`, `documentation`): Indicates the type of issue.
+    - `milestone/...`: Specifies the target release for the issue.
+    - `priority/...`:  Indicates the urgency of the issue.
+    - `good-first-issue`:  Marks issues suitable for first-time contributors.
+    - `backported/...` (e.g., `backported/v0.1.2`): Use this label for PRs that backport changes from the `main` branch to a release branch (v0.1.2).
```

Comment:
Add above backported:

- `candidate/...`: (e.g., `candidate/v0.1.2`)  Identifies PRs in the `main` branch as candidates for backporting or cherry-picking to a release branch (e.g., `v0.1.2`). The specific method will be determined during the porting process.